modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/jaxrs/context/provider/APIApplicationContextProvider.java

```diff
@@ -56,7 +56,7 @@ public APIApplication createContext(Message message) {
 		}
 	}
 
-	private APIApplication _getApiApplication(
+	private APIApplication _fetchApiApplication(
```

Comment:
Not sure if this method is needed as it is only called once

Reply:
Has been inlined